default/stringtables/en.txt

```diff
@@ -207,6 +207,63 @@ Unowned
 NOWHERE
 Cannot be produced
 
+################################## 
+# FORMAT_LIST items are for lists of the form
+# FORMAT_LIST_HEADER FORMAT_LIST_x_ITEMS
+# i.e.
+# There is none.
+# There is only: banana
+# There are: apple and banana
+##################################
+
+FORMAT_LIST_DEFAULT_PLURAL_HEADER
+There are:
+
+FORMAT_LIST_DEFAULT_SINGLE_HEADER
+There is one:
+
+FORMAT_LIST_DEFAULT_EMPTY_HEADER
+There is none.
+
+FORMAT_LIST_DEFAULT_DUAL_HEADER
+There are:
+
+FORMAT_LIST_0_ITEMS
+%1%
+
+FORMAT_LIST_1_ITEMS
+%1% %2%.
+
+FORMAT_LIST_2_ITEMS
+%1% %2% and %3%.
+
+FORMAT_LIST_3_ITEMS
```

Comment:
Rather than dealing with all these list formatting issues, why not just have a space-separated list with no extra special formatting required to add an extra item to the end? works well enough in the pedia...